functions/helloworld/index.js

```diff
@@ -19,10 +19,9 @@
 // [START functions_helloworld_http]
```

Comment:
Do you think something like this would be more clear to future contributors? 
```suggestion
// [START functions_helloworld_http]
// [START functions_helloworld_get]
const functions = require('@google-cloud/functions-framework');
// [END functions_helloworld_get]
const escapeHtml = require('escape-html');
// [END functions_helloworld_http]
```

Reply:
Thanks! Made a similar change. 👍